src/modules/assets/assets.service.ts

```diff
@@ -310,7 +310,7 @@ export class AssetsService {
         this.assetsRepository.save(update);
 
         this.eventEmitter.emit(
-          AssetHistoryEventType.ASSET_TRANSFERRED,
+          AssetHistoryEventType.OFFERED_FROM,
```

Comment:
This change doesn't really make sense to me. The smart contract event name is `'IdentityTransferred'` so don't understand why we would emit an `OFFERED_FROM` event. @DawidGil-jlabs could you please explain?